Documentation/ApiOverview/Database/BasicCrud/_MyDeleteRepository.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Vendor\MyExtension\Domain\Repository;
```

Comment:
```suggestion
namespace Vendor\MyExtension\Domain\Repository;
```

A while ago we had the discussion that Namespace "Vendor" and directory "vendor" are prone to mix up. Since then I have switched to MyVendor as default vendor namespace. I think we have no general role, but what you think?